lib/checkPullRequestLabels.js

```diff
@@ -41,6 +41,9 @@ const assignAllCodeowners = async (context) => {
    * @type {import('probot').Octokit.PullsGetResponse} pullRequest
    */
   const pullRequest = context.payload.pull_request;
+  // Wait for github finding reviewers.
+  await utilityModule.sleep(30000);
```

Comment:
Hmm, lets put a bit less, I think 10 seconds should be enough, 30 seconds wait can make the oppiabot timeout and not finish.